### PR TITLE
Feature/mentor application

### PR DESCRIPTION
**Pull request by danielaescutia07**
## Description

https://www.loom.com/share/f2ba8df6e143420cba1c0654c304a283

This PR improves questions as well as options to choose from in lieu of text boxes on the Mentor Form.  This modification provides a preferable user experience by supplying the user with multiple options. This change will allow DS to utilize the data for mentor-mentee pairing suggestions.

## Contributors

@danielaescutia07 
@tinaxgao 
@AnthonyMillerGit 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/RoleSignup/Applications/Mentee.js [hunk 1]
```diff
@@ -1,20 +1,24 @@
 import React, { useState } from 'react';
```
Comment (tinaxgao) [inline src/components/pages/RoleSignup/Applications/Mentee.js:1]: Please don't include the Mentee file in this PR

File: src/components/pages/RoleSignup/Applications/Mentor.js [hunk 2]
```diff
@@ -47,6 +52,11 @@ const Mentor = () => {
     });
   };
 
+  function handleChangeStates(value) {
```
Comment (tinaxgao) [inline src/components/pages/RoleSignup/Applications/Mentor.js:55]: see my updated mentee file for examples for this & all the onChange functions

**Review by tinaxgao — CHANGES_REQUESTED** [review]

File: src/components/pages/RoleSignup/Applications/Mentor.js [hunk 3]
```diff
@@ -200,103 +271,85 @@ const Mentor = () => {
                     <Input placeholder="Your answer" />
```
Comment (tinaxgao) [inline src/components/pages/RoleSignup/Applications/Mentor.js:271]: You need to add an onChange here to record the value

**Review by tinaxgao — CHANGES_REQUESTED** [review]

**Review by tinaxgao — APPROVED** [review]
great changes to update mentor form to accept more relevant data for DS. Looks consistent with Mentee form.

**Review by tristanharrower — APPROVED** [review]
The changes satisfy the DS requirements and are written effectively.  Good Job!

**Review by AnthonyMillerGit — COMMENTED** [review]
the text boxes being changed was great. just needs to be a bit more responsive on mobile

**Pull request merged**